publisher/moloco.js

```diff
@@ -65,7 +67,16 @@ MolocoSDK.prototype.requestAd = function (adDiv) {
     if (this.readyState === 4 && this.status === 200) {
       switch (context.adType) {
         case AdType.BANNER:
-          context.renderAd(xhr.responseText);
+          if (context.isLegacy) {
+              context.renderAd(xhr.responseText);
+          } else {
+              bannerAd = JSON.parse(xhr.responseText);
+              if (context.useRedirectUrl) {
+                  context.renderAdWithRedirectUrl(bannerAd);
```

Comment:
If renaming use_redirect_url, this name also needs to be revised.

Reply:
This function name `renderAdWithRedirectUrl` needs to be revised as it uses direct landing URL, not redirect URL? :)